microsoft-edge/devtools-guide-chromium/javascript/source-maps.md

```diff
@@ -31,26 +31,35 @@ Keep your client-side code readable and debuggable even after you combine, minif
 *   Only use pre-processors capable of producing Source Maps.
 *   Verify that your web server is able to serve Source Maps.
 
-<!--todo: add link to preprocessors capable of producing Source Maps when section is available -->
-<!--[]: /web/tools/setup/setup-preprocessors?#supported_preprocessors ""  -->
+<!--
+no longer in original file:
```

Comment:
I'm not sure I understand what you mean by "no longer in original file". Can you explain what this line is meant for?